acm/overlays/moc/infra/managedclusters/jerry.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: jerry
+  labels:
+    cluster.open-cluster-management.io/clusterset: argocd-managed
+    authdeployment: primary
```

Comment:
@HumairAK looking at https://www.operate-first.cloud/apps/content/acm/manage_your_teams_ocp_clusters_via_acm.html#steps 

> However, to configure your cluster to enable auth via our IDP [AuthRealm](https://github.com/operate-first/apps/blob/master/acm/overlays/moc/infra/identitatem/authrealms/primary.yaml) you will need to add the label authdeployment: primary to your clusters’ ManagedCluster resource.

Is this label sufficient to add this cluster to identitatem and configure SSO?

Reply:
Yes, this is sufficient. The rest is taken care of.